src/main/steps/respondent/task-list/utils.ts

```diff
@@ -275,6 +275,40 @@ export const getRespondentPartyDetailsCa = (userCase: Partial<CaseWithId>, userI
   return undefined;
 };
 
+export const getResponseStatus = (userCase: Partial<CaseWithId> | undefined, userId: string): SectionStatus => {
+  const respondent1 = userCase?.respondents?.find(respondent => {
```

Comment:
please change the const to respondent and inner argument of find method to _respondent

Reply:
Yes this is taken care.